folly/io/async/NotificationQueue.h

```diff
@@ -225,7 +225,7 @@ class NotificationQueue {
     : eventfd_(-1),
       pipeFds_{-1, -1},
       advisoryMaxQueueSize_(maxSize),
-      pid_(getpid()),
+      pid_((pid_t)getpid()),
```

Comment:
With the windows implementation of pthread, it defines `pid_t` as a `void*`, and `getpid()` returns an `int`. MSVC doesn't like this implicit construction of a pointer from a raw integer and produces an error. Explicitly casting it solves the problem.